redwood-layout-composeui/src/main/kotlin/app/cash/redwood/layout/composeui/utils.kt

```diff
@@ -68,7 +69,20 @@ internal fun CrossAxisAlignment.toAlignSelf() = when (this) {
   else -> throw AssertionError()
 }
 
-internal fun Padding.toSpacing() = Spacing(start.toDouble(), end.toDouble(), top.toDouble(), bottom.toDouble())
+internal fun Padding.toSpacing(context: Context): Spacing {
```

Comment:
I'd like to pull a lot of the methods from this file into a `redwood-layout-common` module so I can stop duplicating code. It'll be published, but it'll be considered private API that can change at any time. What do you think?

Reply:
We could try to set up repackaging so it's not actually published but just locally shared. Failing that, we could annotate everything with an `@OptIn` annotation to convey that it's unstable API. All of that can come later, though. Extracting a published module to eliminate duplication seems fine.

Reply:
Sounds good, I'll try sharing it locally + repackaging. Do you know if we've done something similar previously in any of Square/Cash App's repos?

Reply:
I've only ever done it for a jar-producing module. Since the ones we want to use it on are Android modules that complicates things outside the realm of what I've done in the past.

Here's the jar one: https://github.com/cashapp/redwood/blob/b8aac3644337e557d434d0dbb5e6dba90c324bec/treehouse/compose/compiler/build.gradle